### ISSUE TITLE
Broken with Legacy Edge

### ISSUE THREAD

**Issue by patrickberkeley** [machty/ember-concurrency-decorators#65]
Legacy (non-chromium based) Edge is broken on this line: https://github.com/machty/ember-concurrency-decorators/blob/70265639f1db69c1d419312675e44e6e483bc12d/addon/index.ts#L165 with the error `Expected identifier, string or number`. We are not using `ember-concurrency-decorators` directly, but via v4.0.0-beta.6 of `ember-power-select`: https://github.com/cibernox/ember-power-select/blob/v4.0.0-beta.6/addon/components/power-select.ts#L542. The error completely prevents the ember app from loading.

**Comment by aatauil** [comment]
@patrickberkeley Did you found any solution for this ?

**Comment by patrickberkeley** [comment]
@aatauil yes, we stopped using ember-concurrency and ember-concurrency-decorators altogether.
